packages/apps/app-dashboard/src/utils/user-dashboard/approvalStateStorage.ts

```diff
@@ -0,0 +1,58 @@
+import { IRelayPKP } from '@lit-protocol/types';
```

Comment:
Please use the [useLocalStorage](https://usehooks-ts.com/react-hook/use-local-storage) hook instead of implementing localstorage integration imperatively here -- we've started re-implementing that hook every time we manage some json state in localstorage :)